### ISSUE TITLE
adding POV 

### ISSUE THREAD

**Issue by illu10** [Riqorm/PlayHomeVR#8]
As I understand there is no POV (girl char following your view with head or eyes) in PlayHome?
Is there a chance to add it? (like it was in HoneySelect, curious if it needs built in suppport or can be done from plugin)
Afair there is a Changer plugin which claims POV support for PlayHome (but seems it doesn't work at least for me in VR)

Thank you!

**Comment by Riqorm** [comment]
I'm not sure about your definition of PoV (Point of View).
If you mean that the girl should look at you (you being the headset), then yes that is in already and explained in the changelog as well as the manual (use the playtool, press trigger).
If you mean you want to take the Point of View of the impersonated person, so move the cam the way this person moves, then this is not in. But it's usually a very bad idea in VR to have someonething else than your own movement change the cam position, as it will induce VR sickness (esp with the grinding motions in PlayHome)

**Comment by illu10** [comment]
Yes, I mean "look at you". Shall try to use controllers. Basically I use air mouse with keyboard. Is there a shortcut to toggle this mode maybe? Thank you

**Comment by agostgi** [comment]
Using the trigger I can get the "look at you" to work in the scenes but not in the character editor. Is it meant to be working in character editor? I've also tried setting the "Gaze" of both head and eyes to Camera, but it doesn't work. By the way, I appreciate the work you did in bringing the post-processing effects into VR.

**Comment by masondunne** [comment]
Can you add a keyboard or X360 controller bind to this for those of us that don't have VR controllers?

**Comment by plasticfant** [comment]
For those not having success with gamechanger, use F5 after pressing P. It blocks rotating of the camera and works quite well in my opinion. Not perfect, of course.

**Comment by ghost** [comment]
@plasticfant You can use Ctrl-x and Ctrl-shift-x for impersonation. The VR plugin does support it currently, though there are some issues with it (#21). You can configure whether or not to allow females as target in `VRSettings.xml`: `<ImpersonationTarget>`. You can sort of work around its quirks by looking to one side and recentering (F12).

@Riqorm It's probably safe to close this as the requested "look at you" feature is already implemented (this issue isn't actually about POV/impersonation).
